### DISCUSSION TITLE
what font is used in the example gifs on the main page

### DISCUSSION THREAD

**Discussion by bsdmp** [joaotavora/eglot#1465] [Q&A]
What font is used in the gifs on https://github.com/joaotavora/eglot? For example on this one https://github.com/joaotavora/eglot/blob/master/gif-examples/eglot-xref-find-definition.gif (they're all the same).

**Comment by rrudakov** [comment]
Looks like [terminus](https://terminus-font.sourceforge.net/shots.html)

**Comment by joaotavora** [comment]
either that or `misc-fixed`.  I don't remember any more
